scripts/start_local_network.sh

```diff
@@ -38,16 +23,17 @@ killall nym-sfw-provider
 echo "Press CTRL-C to stop."
 echo "Make sure you have started nym-directory !"
 
-cargo build --release --manifest-path nym-client/Cargo.toml --features=local
-cargo build --release --manifest-path mixnode/Cargo.toml --features=local
-cargo build --release --manifest-path sfw-provider/Cargo.toml --features=local
+cargo build --manifest-path nym-client/Cargo.toml --features=local
```

Comment:
Why are we not building it in release anymore?

Reply:
The only downside of switching to Rust has been the slow compilation times. We can build this in release, but you may lose a team member to suicide. 

I can't see any particular reason that most people would want this built in release mode. Platform developers want the fastest compilation times. External developers who want to run a testnet for their own local development purposes (which is what I was doing yesterday) probably also just want it to work as quickly as they can get their own code running. 

If we needed to benchmark a local network, that's probably something we could handle pretty easily, and it's a less frequent task.